src/matlab2tikz.m

```diff
@@ -6329,6 +6327,14 @@ function printAll(m2t, env, fid)
     end
     opts = opts_append(opts, key, value);
 end
+function opts = opts_addOpts(m2t, opts, key, subOpts, sep)
```

Comment:
So you will hate me for that, but can we rename it to
```
opts_addSubOpts
```
Because this is generally a suboption of some axis option. So we can differentiate, when we have curly braces around

Also i cannot think of any case where we would not use ',' as a separator.